ambiegenvae/common/test_vae.py

```diff
@@ -0,0 +1,101 @@
+
+import torch
+import torch.utils.data
+
+from torchvision import transforms
+
+import numpy as np
+from ambiegenvae.generators.kappa_generator import KappaRoadGenerator
+from ambiegenvae.generators.abstract_generator import AbstractGenerator
+from ambiegenvae.common.train_vae import VecVAE, Denormalize1D, Normalize1D, ToTensor1D
+
+
+def load_model(epoch, model, path='.//'):
+
+    # creating the file name indexed by the epoch value
+    filename = path + '/neural_network_{}.pt'.format(epoch)
+
+    # loading the parameters of the saved model
+    model.load_state_dict(torch.load(filename))
```

Comment:
Since we cannot guarantee an Nvidia-GPU environment for the evaluation (and we did not announce that one would be available, this line will be changed to `model.load_state_dict(torch.load(filename, map_location=torch.device('cpu')))` to enable CPU runs.

Reply:
Sure, if the gpu is not available, "cpu" will be used ( (with this code: device = torch.device("cuda" if torch.cuda.is_available() else "cpu") ).